source/features/repo-avatars.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	// Icon for public but not template/fork/etc. repos
+	const icon = await elementReady('#repository-container-header .octicon-repo');
+	if (!icon) {
+		return;
+	}
+
+	const author = select('#repository-container-header [rel=author]')!;
+	const username = author.textContent!;
+	const alt = `@${username}`;
+	const fallbackSrc = (pageDetect.isEnterprise()
+		? `/${username}.png`
+		: `https://avatars.githubusercontent.com/${username}`) + '?size=48';
+	const src = select(`img[alt="${alt}"]`)?.src ?? fallbackSrc;
+
+	const avatar = (
+		<img
+			className="avatar mr-2"
+			src={src}
+			width="24"
+			height="24"
+			alt={alt}
+		/>
+	);
+
+	if (author.dataset.hovercardType === 'user') {
```

Comment:
If you use getRepo, this can be replaced by a select.exists, or maybe we can add this check to url-detections as isOrganizationRepo

Reply:
We already have `isOrganizationRepo()` 😃!